_data/tools.yml

```
@@ -14,8 +67,201 @@
+# there are lots more from Trino Fest, Trino Summot .. need to figure out how many we are happy to have.. 
+- name: Apache Ignite
+  anchor: apache-ignite
+  category: data-source
+  logo: /assets/images/logos/apache-ignite.png
+  logosmall: /assets/images/logos/apache-ignite-small.png
+  description: |
+    Apache Ignite is a distributed database for high‑performance applications
+    with in‑memory speed. It scales across memory and disk across multiople
+    machines without compromise.
+
+    Use an Apache Ignite database as a data source in Trino by configuring a
+    catalog with the Apache Ignite connector.
+  links:
+    - urltext: Apache Ignite
+      url: https://ignite.apache.org/
+    - urltext: Apache Ignite connector documentation
+      url: https://trino.io/docs/current/connector/ignite.html
+    - urltext: Interview about the Ignite connector from Trino Community
+        Broadcast 46
+      url: https://trino.io/episodes/46.html
+- name: Apache Kafka
+  anchor: apache-kafka
+  category: data-source
+  logo: /assets/images/logos/apache-kafka.png
+  logosmall: /assets/images/logos/apache-kafka-small.png
+  description: |
+    Apache Kafka is an open-source distributed event streaming platform used by
+    thousands of companies for high-performance data pipelines, streaming
+    analytics, data integration, and mission-critical applications.
+
+    Use an Apache Kafka topic stream as a data source in Trino by configuring a
+    catalog with the Apache Kafka connector.
+  links:
+    - urltext: Apache Kafka
+      url: https://kafka.apache.org/
+    - urltext: Apache Kafka connector documentation
+      url: https://trino.io/docs/current/connector/kafka.html
+- name: Apache Kudu
+  anchor: apache-kudu
+  category: data-source
+  logo: /assets/images/logos/apache-kudu.png
+  logosmall: /assets/images/logos/apache-kudu-small.png
+  description: |
+    Apache Kudu is an open source distributed data storage engine that makes
+    fast analytics on fast and changing data easy.
+
+    Use an Apache Kudu data storage as a data source in Trino by configuring a
+    catalog with the Apache Kudu connector.
+  links:
+    - urltext: Apache Kudu
+      url: https://kudu.apache.org/
+    - urltext: Apache Kudu connector documentation
+      url: https://trino.io/docs/current/connector/kudu.html
+- name: Apache Phoenix
+  anchor: apache-phoenix
+  category: data-source
+  logo: /assets/images/logos/apache-phoenix.png
+  logosmall: /assets/images/logos/apache-phoenix-small.png
+  description: |
+    Apache Phoenix enables OLTP and operational analytics in Hadoop for low
+    latency applications by combining the best of both worlds:
+
+    * The power of standard SQL and JDBC APIs with full ACID transaction
+      capabilities and
+    * The flexibility of late-bound, schema-on-read
+      capabilities from the NoSQL world by leveraging HBase as its backing store
+
+    Use a Apache Phoenix data lake as a data source in Trino by configuring a
```

Phoenix is not a data lake last time I checked 😁


```suggestion
    Use a Apache Phoenix key value store as a data source in Trino by configuring a
```